### ISSUE TITLE
Clarifications to 'Get Started!' section of documentation, based on errors encountered

### ISSUE THREAD

**Issue by Ram-N** [ericmjl/pyjanitor#532]
## Clarifications to the `Get Started!` section of contributing.html

All: It took me a surprisingly long time to get the `pyjanitor` development environment set up on Windows 10. I have a couple of questions and an offer to add some clarifications

## Relevant Context

- [Contributing.html](https://pyjanitor.readthedocs.io/contributing.html)
- What follows refers to the [Get Started!](https://pyjanitor.readthedocs.io/contributing.html#get-started) section
### 1. Running make on Windows
I got stuck while trying to run make. The documentation is helpful and instructed me to run:

     $ conda install -c defaults -c conda-forge make

This worked fine. However, even though `make` was installed, I got the 'bash: make not found' error. 

*Question:* How to follow up after conda installing make? Do I need to add it explicitly to $PATH? The documentation does not explain this part.

After a lot of searching, I found one fix that let me proceed. It had to do copying with MinGW, 
which has a `make` in one of the bin directories. (I can add a note in docs to this for other Windows developers who will follow. Not sure how typical this is.)

Finally, got `make install` to work fine.

### 2. Running `make docs`

Next, I was hit with an error saying modules were not found. (But `make install` had worked fine.) I was following the instructions verbatim, but there was no mention of activating the conda environment.

Once I ran `source activate pyjanitor-dev` things worked fine when I *then* ran `make docs`

If my understanding is correct, source activate is needed prior to running `make docs` 

Based on my experience, I'd like to make a few very minor modifications to the 'Getting Started' section of the documentation to make it easier for future developers of pyjanitor.

Please let me know if this is something that would be relevant. Also, I am not sure if my understanding of the steps is correct.

Or perhaps my experience was not typical, in which case we shouldn't modify the Getting Started instructions.

**Comment by ericmjl** [comment]
@Ram-N thank you for pinging in! :smile: Your feedback is valuable; the Windows-specific sections have been the result of many other contributors who use Windows (I do not, personally) wrestling, and I'd love to have your contribution added.

The best places might be as an [RST admonition](http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions) right in the context of where you faced those issues. Would you like to work on this? If so, I will assign this issue to you and mark it as taken, and we can take it from the PR.

**Comment by Ram-N** [comment]
Yes, please assign it to me. (I've never worked on an RST admonition before, but I'm happy to try it. Will issue a PR if I get it done. Thanks

**Comment by jk3587** [comment]
@Ram-N Good to see another developer on windows!

Regarding installing make:
`conda install -c conda-forge -n base make`
This should install make to your base environment script folder and should allow you to run make any time you have the base environment activated.

Regarding `make install`:
Reading the Makefile, it shows:
```
ACTIVATE=source activate pyjanitor-dev

---Some other recipes---

install:
	@echo "Creating Conda environment..."
	conda env create -f environment-dev.yml

	@echo "Installing PyJanitor in development mode..."
	$(ACTIVATE) && python setup.py develop

	@echo "Registering current virtual environment as a Jupyter Python kernel..."
	$(ACTIVATE) && python -m ipykernel install --user --name pyjanitor-dev --display-name "PyJanitor development"
```
`ACTIVATE=source activate ENV_NAME` doesn't really work on vanilla Windows (unless you use some sort of bash terminal like MinGW or Cmder)
Changing to `ACTIVATE=conda activate pyjanitor-dev` should let you activate your environment and get the rest of the script working.

Another solution is to install WSL and skip all these Windows specific issues.

@ericmjl Perhaps changing the `ACTIVATE` variable in the Makefile to use `conda activate` instead of `source activate` would allow `make install` to work for Windows users along with Mac/Nix?

**Comment by Ram-N** [comment]
@jk3587 Thank you for jumping in. Much appreciated.

You said:

> Regarding installing make:
> `conda install -c conda-forge -n base make`
> This should install make to your base environment script folder and should allow you to run make any time you have the base environment activated.

For whatever reason, when I ran `make install` right after running the conda install command above, I got the `make not found` error. Since I am planning to change the documentation, I want to be doubly sure.

Question: Will running 
`conda install -c conda-forge -n base make` install `make` to `~/Anaconda3/Library/bin`
or will it go to the `Scripts` directory? 
In my case I found make in `Library/bin`, but not in `Scripts`

Thank you for your help.

**Comment by jk3587** [comment]
@Ram-N My apologies, it is installed in the `Library/bin` folder.
Also, does the default anaconda installation append the anaconda folder to the windows path? If not, that could also be an issue.
I recall there's a shortcut that is created during anaconda installation called "Anaconda Prompt" which opens up a command prompt with base environment already activated.
A way of testing this out would be:

1. Open up a regular command prompt
2. Try `conda activate base` or `activate base`
3. Then try if just `make` works.  If it works, you'll get the error message like `make: *** No targets specified and no makefile found.  Stop.`

**Comment by Ram-N** [comment]
Thanks, @jk3587 I will try adding an "RST admonition" or two, as @ericmjl suggested.

1. Yes, from the Anaconda Prompt, make works.
2. However, when I installed Anaconda, I did not check the box for altering the PATH variable. 
(see this image)
![image](https://user-images.githubusercontent.com/737241/63139793-8b5ebd00-bfad-11e9-80ad-f6814870eddf.png)
3. At the time, I didn't know that conda was installing `make` to `~/Anaconda3/Library/bin`

Thanks again.
